pkg/statistics/component.go

```diff
@@ -139,6 +140,13 @@ func (ec *component) GetStatisticsAuthenticationsLog(ctx context.Context, realmN
 
 	var res []api.StatisticsConnectionRepresentation
 
+	ok, _ := regexp.MatchString(api.RegExpTwoDigitsNumber, max)
+
+	if !ok {
```

Comment:
Go allows you to write this on a single line ;-)
if ok, _ := regexp.MatchString(api.RegExpTwoDigitsNumber, max); !ok {
}

Reply:
Done